MANIFEST.in

```
@@ -1,5 +1,16 @@
 include *.rst
 recursive-include examples *
+
 include LICENSE
 include README.rst
 include pyproject.toml
+
+prune benchmarks
+prune .github
+prune alphacsc/other/sporco
```

I think we can completely remove the `other` folder as it is not part of our code.

```suggestion
prune alphacsc/other
```

Maybe we can add a note on the examples using `other` to warn that this is not part of the package.